prescription/test/integrity.go

```diff
@@ -7,112 +7,109 @@ import (
-				Units: pointer.FromString("Units/hour"),
-				Value: pointer.FromFloat64(0.2),
+		},
+		GlucoseSafetyLimit: pointer.FromFloat64(0.7),
+		InsulinModel:       pointer.FromString("rapidAdult"),
+		InsulinSensitivitySchedule: &pump.InsulinSensitivityStartArray{
+			&pump.InsulinSensitivityStart{
+				Amount: pointer.FromFloat64(0.8),
+				Start:  pointer.FromInt(456789),
 			},
-			BolusAmountMaximum: &pump.BolusAmountMaximum{
-				Units: pointer.FromString("Units"),
-				Value: pointer.FromFloat64(0.2),
+			&pump.InsulinSensitivityStart{
+				Amount: pointer.FromFloat64(0.9),
+				Start:  pointer.FromInt(9876),
 			},
-			PumpID: pointer.FromString("1234567890"),
-			CgmID:  pointer.FromString("1234567890"),
 		},
-		Calculator: &prescription.Calculator{
-			Method:                        pointer.FromString("weight"),
-			RecommendedBasalRate:          pointer.FromFloat64(0.1),
-			RecommendedCarbohydrateRatio:  pointer.FromFloat64(1.0234),
-			RecommendedInsulinSensitivity: pointer.FromFloat64(0.89),
-			TotalDailyDose:                pointer.FromFloat64(2.33),
-			TotalDailyDoseScaleFactor:     pointer.FromFloat64(1.02),
-			Weight:                        pointer.FromFloat64(81.33),
-			WeightUnits:                   pointer.FromString("kg"),
+		BasalRateMaximum: &pump.BasalRateMaximum{
+			Units: pointer.FromString("Units/hour"),
+			Value: pointer.FromFloat64(0.2),
 		},
-		Training:                pointer.FromString("inPerson"),
-		TherapySettings:         pointer.FromString("transfer"),
-		PrescriberTermsAccepted: pointer.FromBool(true),
-		State:                   "submitted",
+		BolusAmountMaximum: &pump.BolusAmountMaximum{
+			Units: pointer.FromString("Units"),
+			Value: pointer.FromFloat64(0.2),
+		},
+		PumpID: pointer.FromString("1234567890"),
+		CgmID:  pointer.FromString("1234567890"),
+	},
+	Calculator: &prescription.Calculator{
+		Method:                        pointer.FromString("weight"),
+		RecommendedBasalRate:          pointer.FromFloat64(0.1),
+		RecommendedCarbohydrateRatio:  pointer.FromFloat64(1.0234),
+		RecommendedInsulinSensitivity: pointer.FromFloat64(0.89),
+		TotalDailyDose:                pointer.FromFloat64(2.33),
+		TotalDailyDoseScaleFactor:     pointer.FromFloat64(1.02),
+		Weight:                        pointer.FromFloat64(81.33),
+		WeightUnits:                   pointer.FromString("kg"),
 	},
-	CreatedUserID: "1234567890",
+	Training:                pointer.FromString("inPerson"),
+	TherapySettings:         pointer.FromString("transfer"),
+	PrescriberTermsAccepted: pointer.FromBool(true),
+	State:                   "submitted",
 }
 
 // ExpectedHash is the expected integrity hash for the values above
-const ExpectedHash = "994c84b67bbace6721fc490e2ede422a88c75b21b20d8d9480d3d85b726ce226dc1f035e9961e604e4c7b230459bddb1950df1e397b3ff518b9ce07c32bc1327"
+const ExpectedHash = "d9b021a9eba11c6edfc433e8569aaa170e41736282c652bc27831cc68d9ed80ff774ee478310098c92d58ecdb6f50b1c1a7f5a0342e3bf2dfc5cc5414b9cc69b"
```

Comment:
`IntegrityAttributesExpectedHash` doesn't add a lot of context where this has is used. You can't communicate with the variable name only that the expect hash is specifically for those values above.

Reply:
Given that the variable above is called IntegrityAttributes, I'd say it does, but that's fine. Keep in mind too, the "values above" is a relative thing, if someone where to accidently move things around, that might no longer be true. *shrug* Just trying to prevent issues later on.